README.md

```diff
@@ -0,0 +1,21 @@
+# Docker Proxy for ghcr.io
+
+This hosts the Dockerfile for building an nginx docker image that will proxy traffic to ghcr.io.
+
+It also caches docker layers, and is RO - no uploads allowed.
```

Comment:
My preference is to avoid abbreviations where possible - could we have `read-only` instead of `RO`? Or could rewrite to be more specific:
 "It caches docker layers. It also only proxies GET requests, which allows docker image downloads, but not uploads."